SEED-platform/seed
# 1525 view history

#### Any background context you want to provide?
Somewhere along the way, the inventory detail view stopped working when there were merged records. The database was okay, just not the view code.

#### What's this PR do?
Moves the get_history command onto the PropertyState and TaxLotState classes. Wrote more tests to verify that the methods do what is expected based on the example-data-* test files.

#### How should this be manually tested?
Import example-data-properties and then example-data-properties-small-changes file and make sure that the inventory detail view looks correct.

#### What are the relevant tickets?
#1525 
#1527 

#### Screenshots (if appropriate)
#### Definition of Done:
